### ISSUE TITLE
[FR] web/*.json 

### ISSUE THREAD

**Issue by rawgni** [PokemonGoF/PokemonGo-Bot#5084]
### Short Description

with web/*.json giving user a lot of problem. Shouldn't we stop using it?

### Possible solution

One of the following:
1. config to allow user to disable the ui component, ie. `enable_ui`
If enable_ui is false, we don't do bother reading/creating/updating the web/* files.
Most people probably run the bot headless.

2. db.
we have a db that store the inventories, and other logs.
the web component should talk to the db directly.

3. socket.
use the socket to get information

### How it would help others

Less Issues ( i guess ? )

**Comment by Gobberwart** [comment]
Yes. Yes we should. Websocket would be ideal, but otherwise db is good. JSON files are a pain in the a**.

**Comment by oralunal** [comment]
@rawgni I'm talking about this these days, I'm developing a web ui but need more help from python devs.

Forexample I'm updating map location etc. via socket but if user select PolylineWalker then bot doesn't emit position_update. These days I'm talkin about this onyl.

**Comment by oralunal** [comment]
#5035

**Comment by Gobberwart** [comment]
@oralunal you're developing ANOTHER web ui?

**Comment by mmixx** [comment]
I modified the main.js of the current web ui and got the websocket working and did some fix on it's layout and some bugs I've encountered... Like when viewing your list of pokemons, it won't show when you have an egg being incubated using the breakable incubator and added a breakdown of the eggs you currently have...

Am currently working on fixing the **Actions** when viewing your pokemons when `actionsEnabled` is set to `true`...
